client/src/components/GAINZ.jsx

```diff
@@ -1,12 +1,30 @@
 import React from 'react';
-
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
```

Comment:
If you're using React Router...

Look into React.Lazy, rather than importing everything on page load